query-node/build.sh

```diff
@@ -16,7 +16,6 @@ set +a
 
 yarn query-node:build
 yarn mappings:build
-ln ../types/augment/all/defs.json mappings/lib/generated/types/typedefs.json
```

Comment:
The defs.json is mounted in the volume in docker-compose, but maybe I overlooked the need to actually also have this file copied in-case the processor is run outside of a docker container?

Reply:
Copy of definitions file has been added as part of building mappings. That way, types should be available both in docker and locally run processor.